src/main/java/net/kodehawa/mantarobot/commands/MoneyCmds.java

```diff
@@ -735,4 +742,8 @@ private void proceedGamble(GuildMessageReceivedEvent event, Player player, int l
                 .limit(15)
                 .run(MantaroData.conn(), OptArgs.of("read_mode", "outdated"));
     }
+
+    static {
+        PERCENT_FORMAT.setMinimumFractionDigits(1); // decimal support
+    }
```

Comment:
Same here.

Reply:
Here, static is necessary to be able to call NumberFormat#setMinimumFractionDigits(int) only once to apply decimal support to the format.

Reply:
@natanbc thoughts?